lms/resources/_js_config/file_picker_config.py

```diff
@@ -31,6 +33,9 @@ def blackboard_config(cls, context, request, application_instance):
     def canvas_config(cls, context, request, application_instance):
         """Get Canvas files config."""
 
+        if not context.is_canvas:
+            return {}
```

Comment:
I don't think we need this. `context.is_canvas` is checked on the line below (and we probably shouldn't be making any Canvas changes here)